<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Conf_Boosting</class>
 <widget class="QMainWindow" name="Conf_Boosting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>575</width>
    <height>255</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>575</width>
    <height>255</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Configuraci√≥n Boosting</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Boost Type</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Weak Count</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Weight Trim Rate</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>20</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Max Depth</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="use_surrogates">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>60</y>
      <width>141</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Use Surrogates</string>
    </property>
   </widget>
   <widget class="QComboBox" name="boost_type">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>111</width>
      <height>32</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>DISCRETE</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>REAL</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>LOGIT</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>GENTLE</string>
     </property>
    </item>
   </widget>
   <widget class="QSpinBox" name="weak_count">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>60</y>
      <width>111</width>
      <height>32</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>999999</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="weight_trim_rate">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>100</y>
      <width>111</width>
      <height>32</height>
     </rect>
    </property>
    <property name="maximum">
     <double>999999.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.050000000000000</double>
    </property>
    <property name="value">
     <double>0.950000000000000</double>
    </property>
   </widget>
   <widget class="QSpinBox" name="max_depth">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>20</y>
      <width>60</width>
      <height>32</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>999999</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>160</y>
      <width>96</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Accept</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>575</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
